src/main/java/dev/task/dndquest/controller/PlayerController.java

```diff
@@ -1,15 +1,40 @@
 package dev.task.dndquest.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import dev.task.dndquest.model.dto.PlayerRequestDto;
+import dev.task.dndquest.security.authentication.AuthenticationService;
+import dev.task.dndquest.security.jwt.JwtProvider;
+import dev.task.dndquest.service.PlayerService;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/api/player")
+@AllArgsConstructor
 public class PlayerController {
+    PlayerService playerService;
+    AuthenticationService authenticationService;
+    JwtProvider jwtProvider;
 
     @GetMapping
-    public String hello(){
+    public String hello(Authentication authentication, HttpServletRequest request){
         return "hello world!";
     }
```

Comment:
wtf?

Reply:
used it for manual testing. Already deleted.